test/checks/navigation/identical-links-same-purpose-after.js

```diff
@@ -0,0 +1,98 @@
+describe('identical-links-same-purpose-after tests', function() {
```

Comment:
Add a test for when `parsedResource` is missing., and one for when they don't have the same keys.